docs/product-installation-guide/src/main/asciidoc/clustering-ps-configure-domain-proc.adoc

```diff
@@ -82,10 +121,13 @@ endif::PAM[]
             <role name="kie-server" send="true" consume="true" create-non-durable-queue="true" delete-non-durable-queue="true"/>
         </security-setting>
 ---- 
+
+
 . Add the following properties to the `<system-properties>` element:
 +
-* `org.kie.server.persistence.ds`: The JNDI name of your data source. For {PRODUCT}, this is `java:/jboss/datasources/rhpam`
-* `org.kie.server.persistence.dialect`: The hibernate dialect for your database
+* `org.kie.server.persistence.ds`: The JNDI name of your data source. For {PRODUCT}, this is `java:/jboss/datasources/rhpam`.
+* `org.jbpm.ejb.timer.tx" value="true`: Required for clustering timers.
```

Comment:
Please adjust the quotes, I guess it can be something like `org.jbpm.ejb.timer.tx value="true"`